src/EnergyPlus/Psychrometrics.cc

```diff
@@ -803,6 +810,11 @@ namespace Psychrometrics {
             Real64 const C12(-0.14452093e-7); // Coefficient for TKel >= KelvinConvK
             Real64 const C13(6.5459673);      // Coefficient for TKel >= KelvinConvK
             Pascal = std::exp(C8 / Tkel + C9 + Tkel * (C10 + Tkel * (C11 + Tkel * C12)) + C13 * std::log(Tkel));
+
+            // If above 200C, set value of Pressure corresponding to Saturation Temperature of 200C.
+        } else { // Tkel >= 173.15 // Tkel >= KelvinConv // Tkel > 473.15
+            Pascal = 1555073.745636215;
+        }
```

Comment:
@rraustad and I noted that the value below +200°C does not match the evaluation of the equation at +200°C. So align:
* One value if using the ASHRAE method: `1555073.745636215`
* Another value if using the IF97 method  : `  1554671.8682698254`
* (the ASHRAE and IF97 equations give slightly different values at +200°C)